lib/skate_web/router.ex

```diff
@@ -53,6 +53,12 @@ defmodule SkateWeb.Router do
     get("/:provider/callback", AuthController, :callback)
   end
 
+  scope "/auth", SkateWeb do
+    pipe_through([:redirect_prod_http, :accepts_html, :browser, :auth])
```

Comment:
I'm not actually sure if `:ensure_auth` is needed or not?

Reply:
I think we actually do want it because the other plugs used in `SkateWeb.AuthManager.Pipeline` don't do anything if there's no Guardian session information at all, whereas `:ensure_auth` uses `Guardian.Plug.EnsureAuthenticated` which does.

Reply:
[`794a91e` (#2380)](https://github.com/mbta/skate/pull/2380/commits/794a91ee64780421905f77a81687b50dfbee8cf8)